doc/sphinx/proof-engine/tactics.rst

```diff
@@ -903,206 +1057,159 @@ Applying theorems
    type isomorphic to a tuple type and ``apply`` looks recursively whether a
    component of the tuple matches the goal, it excludes components whose
    statement would result in applying an universal lemma of the form
-   ``forall A, ... -> A``. Excluding this kind of lemma can be avoided by
+   ``forall A, … -> A``. Excluding this kind of lemma can be avoided by
    setting the following flag:
```

Comment:
This sentence referred to the `Universal Lemma Under Conjunction` flag which was removed. So this sentence should be removed as well. And the rest of this note should be moved inside the documentation of `apply` next to where it talks about inductive types isomorphic to a tuple.